mobile/src/navigations/main-bottom-tab/styles.ts

```diff
@@ -0,0 +1,16 @@
+import { type BottomTabNavigationOptions } from '@react-navigation/bottom-tabs';
+
+import { Color } from '~/bundles/common/enums/enums';
+
+const bottomTabStyles: BottomTabNavigationOptions = {
+    headerShown: false,
+    tabBarActiveTintColor: '#FFFFFF',
+    tabBarInactiveTintColor: Color.TEXT2,
+    tabBarStyle: {
+        backgroundColor: Color.PRIMARY,
+        height: 97,
```

Comment:
```suggestion
```
Let's make it 50 or just add some paddings. Let's avoid using some magical numbers

Reply:
For example on my phone 97 height looks too big, because I also have android bottom menu